tests/rgbpp/xudt/3-btc-leap-ckb.ts

```diff
@@ -29,6 +33,7 @@ const leapFromBtcToCKB = async ({ rgbppLockArgsList, toCkbAddress, xudtTypeArgs,
       toCkbAddress,
       isMainnet,
       btcTestnetType: BTC_TESTNET_TYPE,
+      btcConfirmationBlocks: 20, // default value is 6
```

Comment:
The normal leaping from BTC to CKB is 6 confirmations, not 20. The test case is for normal leaping not for Seal2earn.

Reply:
> The normal leaping from BTC to CKB is 6 confirmations, not 20. The test case is for normal leaping not for Seal2earn.

https://github.com/ckb-cell/rgbpp-sdk/pull/262/commits/5599bedd9feb23ab8932b3507eec8a405deee357